hubblestack/extmods/modules/fdg.py

```diff
@@ -236,3 +237,17 @@ def _get_top_data(topfile):
             ret.extend(data)
 
     return ret
+
+
+def _get_fdg_file(fdg_file):
+    """
+    Get FDG file path
+    :param fdg_file: Name of fdg file
+    :return: Proper path of fdg file
+    """
+    if not fdg_file:
+        log.warning('fdg.run called without any fdg_file')
+        return None
+    if fdg_file.startswith('salt://'):
+        return fdg_file
+    return 'salt://' + BASE_DIR_FDG_PROFILES + os.sep + fdg_file.replace('.', os.sep) + '.fdg'
```

Comment:
Please check this.
We are restricting fdg files to be in this folder only. I think we currently put fdg files for audit and fdg files for fdg module separately.

Reply:
Wer are only restricting fdg files in this folder when they are passed as file name only. In case complete path is provided (like in audit fdg connector), we are honouring the whole path.